nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -487,11 +503,19 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(IAccessible2* pacc,
 	// Certain objects are never interactive, even if other checks are true.
 	bool isNeverInteractive = parentNode->isHidden||(!isEditable && (isRoot || role == ROLE_SYSTEM_DOCUMENT || role == IA2_ROLE_INTERNAL_FRAME));
 	bool isInteractive = !isNeverInteractive && (isEditable || inLink || states & STATE_SYSTEM_FOCUSABLE || states & STATE_SYSTEM_UNAVAILABLE || isEmbeddedApp || role == ROLE_SYSTEM_EQUATION);
-	// We aren't finished calculating isInteractive yet; actions are handled below.

```

Comment:
This comment should stay here rather than being moved below. This comment relates to `isInteractive`, not `nameIsContent`.